src/generic_core/social.ts

```diff
@@ -702,7 +707,7 @@ function freedomClientToUproxyClient(
   return {
     userId:    freedomClientState.userId,
     clientId:  freedomClientState.clientId,
-    status:    UProxyClient.Status[freedomClientState.status],
+    status:    (<any>UProxyClient.Status)[freedomClientState.status],
```

Comment:
To get rid of a type error, it was reporting that the index type for this was not defined